src/main/java/nl/knaw/dans/migration/core/ExpectedLoader.java

```diff
@@ -54,6 +54,7 @@ public void expectedMigrationFiles(String doi, String[] migrationFiles, FileRigh
       expectedFile.setRemovedDuplicateFileCount(0);
       expectedFile.setTransformedName(false);
       expectedFile.setDefaultRights(datasetRights);
+      expectedFile.setVisibleTo("ANONYMOUS");
```

Comment:
i also want accessibleTo
```expectedFile.setAccessibleTo("ANONYMOUS");```

Reply:
I suppose the embargo-date should also not be set so I'll drop the call to `setDefaultRights`.